src/jobs/quality_gate_js_runner_job.yml

```
@@ -0,0 +1,34 @@
+description: Run frontend tests
```

Maybe this would be a bit more explained?!
```suggestion
description: Run any npm script (like ESLint, TS lint, phrase merge, dead code detection, ...)
```